packages/ui-extensions/src/surfaces/admin/extension-points.ts

```diff
@@ -14,6 +14,60 @@ export interface ExtensionPoints {
     CustomerSegmentationTemplateApi<'admin.customers.segmentation-templates.render'>,
     CustomerSegmentationTemplateComponent
   >;
+
+  // Blocks
+  'admin.product-details.block.render': RenderExtension<
+    StandardApi<'admin.product-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.order-details.block.render': RenderExtension<
+    StandardApi<'admin.order-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.customer-details.block.render': RenderExtension<
+    StandardApi<'admin.customer-details.block.render'>,
+    AnyComponent
+  >;
+
+  // Actions
+  'admin.product-details.action.render': RenderExtension<
+    StandardApi<'admin.product-details.action.render'>,
+    AnyComponent
+  >;
+  'admin.order-details.action.render': RenderExtension<
+    StandardApi<'admin.order-details.action.render'>,
+    AnyComponent
+  >;
+  'admin.customer-details.action.render': RenderExtension<
+    StandardApi<'admin.customer-details.action.render'>,
+    AnyComponent
+  >;
+  'admin.product-index.action.render': RenderExtension<
+    StandardApi<'admin.product-index.action.render'>,
+    AnyComponent
+  >;
+  'admin.order-index.action.render': RenderExtension<
+    StandardApi<'admin.order-index.action.render'>,
+    AnyComponent
+  >;
+  'admin.customer-index.action.render': RenderExtension<
+    StandardApi<'admin.customer-index.action.render'>,
+    AnyComponent
+  >;
+
+  // Bulk Actions
+  'admin.product-index.selection.render': RenderExtension<
```

Comment:
Noticed these don't line up with Elana's PR: https://github.com/Shopify/web/pull/97985

Should be in this format: `'admin.product-index.selection-action.render'`

Reply:
Good catch
I'll update